Automattic/newspack-plugin
# Update the CSS of the Components for more consistency

### All Submissions:






### Changes proposed in this Pull Request:

Small CSS tweaks to all the components to bring them closer to the Calypso components, including:
* Reworked the general font-size/line-height
* Reduced padding of the buttons
* Modified focus behaviour of input/select
* Unified paddings/margins

### How to test the changes in this Pull Request:

1. Run the setup wizard
2. Check the dashboard and all the pages
3. Look at the Components demo page

### Other information:
